### PR TITLE
Add Jiahao's coreteam details

### PR DESCRIPTION
**Pull request by woojiahao**

### CODE REVIEW

**Comment by JingYenLoh** [comment]
Why am I getting asked to review this ???

**Comment by kouyk** [comment]
> Why am I getting asked to review this ???

I thought you're always the one reviewing ;)

**Comment by kokrui** [comment]
@JingYenLoh blame @DrWala

**Comment by DrWala** [comment]
@kokrui Why r u outing me

File: data/coreteam_members.yml [hunk 1]
```diff
@@ -154,3 +154,9 @@ members:
     description: |
       a Computer Science undergraduate interested in programming languages and software engineering. He also enjoys reading, climbing and watching movies.
       Reach out to him at <a href="mailto:ravern@nushackers.org">ravern@nushackers.org</a>.
+
+  - name: Jia Hao
+    description: |
+      a computer science undergraduate who enoys learning and talking about the many sides of computer science (but mostly
```
Comment (kokrui) [inline data/coreteam_members.yml:160]: I'm gonna assume that "enoys" was meant to be "enjoys"

**Review by kokrui — APPROVED** [review]

**Pull request merged**